pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,185 @@
+# Using Spark SQL Streaming with Pub/Sub Lite
+
+The samples in this directory show how to read messages from and write messages to Pub/Sub Lite from an [Apache Spark] cluster created with [Cloud Dataproc] using the [Pub/Sub Lite Spark Connector].
+
+Get the connector's uber jar from this [public Cloud Storage location](gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar). Alternatively, visit this Maven [link](https://search.maven.org/search?q=g:com.google.cloud%20a:pubsublite-spark-sql-streaming) to download the connector's uber jar. The uber jar has a "with-dependencies" suffix. You will need to include it on the driver and executor classpaths when submitting a Spark job, typically in the `--jars` flag. 
+
+## Before you begin
+
+1. Install the [Cloud SDK].
+   > *Note:* This is not required in [Cloud Shell]
+   > because Cloud Shell has the Cloud SDK pre-installed.
+
+1. Create a new Google Cloud project via the
+   [*New Project* page] or via the `gcloud` command line tool.
+
+   ```sh
+   export PROJECT_ID=your-google-cloud-project-id
+   gcloud projects create $PROJECT_ID
+   ```
+   Or use an existing Google Cloud project.
+   ```sh
+   export PROJECT_ID=$(gcloud config get-value project)
+   ```
+
+1. [Enable billing].
+
+1. Setup the Cloud SDK to your GCP project.
+
+   ```sh
+   gcloud init
+   ```
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=pubsublite.googleapis.com,dataproc,storage_component): Pub/Sub Lite, Dataproc, Cloud Storage.
```

Comment:
For all other links you chose not to inline, so please do the same here ie put the actual link below.

Reply:
Done.